helm/daq-config-server/templates/configmap.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "daq-config-server.fullname" . }}-config
+data:
+  config.yaml: |-
+    {{- toYaml .Values.config.logging_config | nindent 4 }}
```

```suggestion
  config.yaml: |-
    {{- toYaml .Values.config | nindent 4 }}
```

This leaves you open to extending config later to add additional keys without having to refactor much more